packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -301,6 +312,9 @@ class Chat extends Widget<Properties> {
       case 'showDayHeaders':
         this._messageList.option(name, value);
         break;
+      case 'typingUsers':
```

Comment:
If action is the same use the following notation
```
case 'showDayHeaders':
case 'typingUsers':
// action
break;
```

Reply:
[1d5739865c023b6c7a90cfcce7a2845805f29860]